integration-tests/src/test/java/com/datastax/dse/driver/api/core/graph/statement/GraphTraversalITBase.java

```diff
@@ -294,7 +294,7 @@ public void should_return_zero_results() {
    */
   @Test
   public void should_return_zero_results_graphson_2() {
-    Assumptions.assumeThat(isGraphBinary()).isFalse();
+    assumeThat(isGraphBinary()).isFalse();
```

Comment:
Side effect of adding the static import in the code that I changed.
I can revert that to avoid impacting unrelated lines.